LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License (ISC)
+
+Copyright (c) 2020 React Native Community
```

Comment:
Potentially this should be 2018 - 2020 since code was contributed throughout all of those year, not sure if it's important though ☺️ 

```suggestion
Copyright (c) 2018 - 2020 React Native Community
```

Reply:
I guess we can add it. I found this template with just a `<year>`, so I didn't think of a range.